.github/pull_request_template.md

```diff
@@ -28,6 +28,15 @@ Please include any screenshots that communicate the visual story of the change t
 Please include any notes that might be helpful for a reviewer to keep in mind while reading the changes.
 -->
 
+### Updated Dependencies
+
+<!--
+Please include any notes that might be helpful for a reviewer to check the dependency changes you might have introduced.
+  - gem version update
+  - new gem introduced
+  - data model update
+-->
+
```

Comment:
Could this be a subsection of `Optional Tasks`, perhaps at the same level of `##### Library-Specific`?

Reply:
Yes! I was on the fence to put it there or not. Now I am not on the fence any longer! THX

Reply:
Or perhaps adding one more sub section under the `Notes`? Somehow this is not a `Optional task` thing. This is more of a add more info thing.

Reply:
Yeah, this isn't really task based. Perhaps it's advice that fits under the Changes section?

Reply:
Moved it under changes and added the default value `None`.